lessons/lesson4.md

```diff
@@ -0,0 +1,145 @@
+## Lesson 4: Ensure Events are published
```

Comment:
IMO this title doesn't reflect the real content.
Should be something like `Consume Events`.

Reply:
done